### ISSUE TITLE
Amiga does not receive any IR data

### ISSUE THREAD

**Issue by Mingo12** [marekhac/HippoPlayerIR#1]
Hi, i need a little help :)
I built everything correctly, but the Amiga does not receive any IR data. I think something with my Arduine is wrong. If I understand correctly, the LED (13) should light up when a button is pressed, even if the Arduino is not connected to the Amiga. But nothing flashes. The sketch was uploaded without error. Maybe it's my Arduino nano, which is probably a cheap clone.

**Comment by marekhac** [comment]
Yes, the LED should flash when you press the key on the remote controller (and wireless IR receiver module got something). I would suggest to check the connection between Arduino and wireless IR receiver module. Is it powered up properly (by +5V from Arduino) ? I think, the cheap clone of Arduino is not a problem (I use them all the time ;P)

**Comment by Mingo12** [comment]
Hi. I found the problem. The IR receiver was defective. Now it works fine. One more note, if someone has similar problems. I have tried different remote controls. Only 1 of 4 is compatible.

Regular Grundig Remote Control: Works
 
Cheap LED Strip Remote: Sends strings with 6 characters. HippoplayerIR obviously expects 8. HippoplayerIR -debug quits with an error.

Amiga CDTV Remote Control: Sends different code every time you press a button, even if you press the same button. Strange  :)

Thanks for this great project. Thumbs up!
